projectEuler/problem2.js

```diff
@@ -0,0 +1,16 @@
+// prompt: by considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms
+function evenFibSum() {
+  let sum = 0;
+  let i = 0;
+  let j = 1;
+  let k = i + j;
+  while (k < 4000000) {
+    i = j;
+    j = k;
+    k = i + j;
```

Comment:
I think I'm following the steps here to calculate each subsequent fibonacci number, but would you please add a comment(s) explaining what is happening with this calculation?

Reply:
I added in the comments. 
Apologies for the wordiness, I was trying to emulate explaining it to someone with no coding experience or exposure to logic puzzles.

Reply:
We will work to pare this down in future PRs but good enough for now